src/booleans.js

```diff
@@ -1,61 +1,62 @@
 function negate(a) {
-  // your code here
+  return !a;
 };
 
 function both(a, b) {
-  // your code here
+  return a && b;
 };
 
 function either(a, b) {
-  // your code here
+  return a || b;
 };
 
 function none(a, b) {
-  // your code here
+  return a == 0 && b == 0;
 };
 
 function one(a, b) {
-  // your code here
+  return !a == b;
 };
 
 function truthiness(a) {
-  // your code here
+  return !!(a); 
 };
 
 function isEqual(a, b) {
-  // your code here
+  return a === b;
 };
 
 function isGreaterThan(a, b) {
-  // your code here
+  return a > b;
 };
 
 function isLessThanOrEqualTo(a, b) {
-  // your code here
+  return a <= b;
 };
 
 function isOdd(a) {
-  // your code here
+  return a % 2 != 0;
 };
 
 function isEven(a) {
-  // your code here
+  return a % 2 == 0;
 };
 
 function isSquare(a) {
-  // your code here
+  return Math.sqrt(a) % 1 === 0;
 };
 
 function startsWith(char, string) {
-  // your code here
+  return char === string[0];
 };
 
 function containsVowels(string) {
-  // your code here
+  return (string.includes("a") || string.includes("e") || string.includes("i") || 
+string.includes("o") || string.includes("u") || string.includes("A") || string.includes("E") || string.includes("I") || string.includes("O") || string.includes("U"))
```

Comment:
As described above, a regex is by far the cleanest solution to this. If you look closely, you should see that the pattern used here is identical to the one described in arrays (except for the missing `^`):
```suggestion
  return strings.filter(string => /[aeiou]/i.test(string));
```